models/OrderDetail.js

```diff
@@ -0,0 +1,25 @@
+/* jshint indent: 1 */
+
+module.exports = function(sequelize, DataTypes) {
+	return sequelize.define('OrderDetail', {
+		order_id: {
+			type: DataTypes.INTEGER(11),
+			allowNull: false,
+			primaryKey: true,
+			references: {
```

Comment:
How come there's generated associations for the foreign keys? Can we still use Eager Loading so that we can pull related data in one promise?

Reply:
Didnt get your point.

I guess this is another way of creating associations. I did confirm through workbench that foreign key was created.
`			references: {
				model: 'UserRole',
				key: 'id'
			}`